server/src/test/java/org/elasticsearch/VersionTests.java

```diff
@@ -186,7 +186,7 @@ public void testMinCompatVersion() {
 
         // from 7.0 on we are supporting the latest minor of the previous major... this might fail once we add a new version ie. 5.x is
         // released since we need to bump the supported minor in Version#minimumCompatibilityVersion()
-        Version lastVersion = Version.V_6_8_0; // TODO: remove this once min compat version is a constant instead of method
+        Version lastVersion = Version.fromString("6.8.0"); // TODO: remove this once min compat version is a constant instead of method
         assertEquals(lastVersion.major, Version.V_7_0_0.minimumCompatibilityVersion().major);
         assertEquals("did you miss to bump the minor in Version#minimumCompatibilityVersion()",
                 lastVersion.minor, Version.V_7_0_0.minimumCompatibilityVersion().minor);
```

Comment:
based on the test failure, we should replace V_7_0_0 with V_8_0_0 in the 2 above lines

Reply:
> based on the test failure, we should replace V_7_0_0 with V_8_0_0 in the 2 above lines

The test still fail even if i replace V_7_0_0 with V_8_0_0

I got the test to pass like this 

```
Version major6x = Version.fromString("6.6.0"); // TODO: remove this once min compat version is a constant instead of method
assertEquals(major6x.major, Version.V_7_0_0.minimumCompatibilityVersion().major);
assertEquals("did you miss to bump the minor in Version#minimumCompatibilityVersion()",
    major6x.minor, Version.V_7_0_0.minimumCompatibilityVersion().minor);
assertEquals(0, Version.V_7_0_0.minimumCompatibilityVersion().revision);
```

And added new assertions for V_7_2_0 and V_8_0_0 : 

> Version lastVersion = Version.V_7_2_0;
>         assertEquals(lastVersion.major, Version.V_8_0_0.minimumCompatibilityVersion().major);
>         assertEquals("did you miss to bump the minor in Version#minimumCompatibilityVersion()",
>             lastVersion.minor, Version.V_8_0_0.minimumCompatibilityVersion().minor);
>         assertEquals(0, Version.V_8_0_0.minimumCompatibilityVersion().revision);

What do you think?